src/sentry/feedback/blueprints/api.md

```diff
@@ -0,0 +1,336 @@
+
+**Authors.**
+
+@cmanallen
+
+**How to read this document.**
+
+This document is structured by resource with each resource having actions that
+can be performed against it. Every action that either accepts a request or
+returns a response _will_ document the full interchange format. Clients may opt
+to restrict response data or provide a subset of the request data. The API may
+or may not accept partial payloads.
+
+## Feedback Index [/organizations/<organization_slug>/feedback/]
+
+- Parameters
+
+  - field (optional, string)
+  - environment (optional, string)
+  - project (optional, string)
+  - sort (optional, string) Default: -timestamp Members
+    - projectId
+    - -projectId
+    - timestamp
+    - -timestamp
+  - statsPeriod (optional, string) - A positive integer suffixed with a unit
+    type. Default: 7d Members
+    - s
+    - m
+    - h
+    - d
+    - w
+  - start (optional, string) - ISO 8601 format (`YYYY-MM-DDTHH:mm:ss.sssZ`)
+  - end (optional, string) - ISO 8601 format. Required if `start` is set.
+  - utc (optiona, 'true') - Whether start/end should use the UTC timezone
+  - cursor (optional, str)
+  - per_page (optional, number) Default: 10
+  - offset (optional, number) Default: 0
+  - query (optional, string) - Search query with space-separated field/value
+    pairs. ie: `?query=browser:Firefox`.
+  - queryReferrer(optional, string) - Specify the page which this query is being
+    made from. Used for cross project query on issue replays page. Pass
+    `queryReferrer=replayUserFeedback` for this query.
+
+### Browse Feedback [GET]
+
+Retrieve a collection of feedback items.
+
+**Attributes**
+
+| Column            | Type             | Description                                         |
+| ----------------- | ---------------- | --------------------------------------------------- |
+| browser           | object           | -                                                   |
+| browser.name      | optional[string] | -                                                   |
+| browser.version   | optional[string] | -                                                   |
+| contact_email     | string           | The contact email of the user writing feedback.     |
+| device            | object           | -                                                   |
+| device.brand      | optional[string] | -                                                   |
+| device.family     | optional[string] | -                                                   |
+| device.model      | optional[string] | -                                                   |
+| device.name       | optional[string] | -                                                   |
+| dist              | optional[string] |                                                     |
+| environment       | string           |                                                     |
+| id                | string           |                                                     |
```

Comment:
could we standardize whether we're calling it `feedback_id` (in the path) or `id` (here)? just so it's less confusing! when we ingest we call it `feedback_id` to store in the database, but i'm currently serializing it as `id` to match what's here.

the pipeline currently is:
POST takes in `event_id` --> STORE IN DB as `feedback_id` --> SERIALIZE as `id` and GET returns to user as` id`

cc @JoshFerge

Reply:
i believe this mirrors a similar problem we have with replay_id. i believe we return `id` in our api responses. i'd be in favor of using `feedback_id` everywhere except for ingest. curious what others think though!